tests/test_other.py

```diff
@@ -1456,7 +1456,6 @@ def test_dynamic_link_with_exceptions_and_assetions(self):
       }
       ''')
 
-    self.node_args += ['--experimental-wasm-threads', '--experimental-wasm-bulk-memory']
```

Comment:
We don't need `@node_pthreads` decorator for this function after this is deleted?

Reply:
This test doesn't seems to need/use threads at all.